hw/top_earlgrey/dv/top_earlgrey_sim_cfgs.hjson

```diff
@@ -44,12 +44,12 @@
              "{proj_root}/hw/ip/rom_ctrl/dv/rom_ctrl_sim_cfg.hjson",
              "{proj_root}/hw/ip/rstmgr/dv/rstmgr_sim_cfg.hjson",
              "{proj_root}/hw/ip/rv_timer/dv/rv_timer_sim_cfg.hjson",
-             "{proj_root}/hw/ip/spi_host/dv/spi_host_sim_cfg.hjson",
```

Comment:
please add this back. I think you probably delete this accidentally when solving the merge conflict.